modules/oauth-customizing-the-oauth-server-URL.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * authentication/configuring-internal-oauth.adoc
+
+[id="customizing-the-oauth-server-url_{context}"]
+= Customizing the internal OAuth server URL
+
+You can customize the internal OAuth server URL by setting the custom hostname and TLS certificate in the `spec.componentRoutes` field of the cluster `Ingress` configuration.
+
+[WARNING]
```

Comment:
@slaskawi @stlaz @yaoli-redhat I added this warning to try capture about finding the certs in the config map, and an example passing in the cert to `oc login`. 

Can you please review this to see if this is accurate/covers the scenarios from https://bugzilla.redhat.com/show_bug.cgi?id=1992431 and https://bugzilla.redhat.com/show_bug.cgi?id=1991448?

Preview: https://deploy-preview-34947--osdocs.netlify.app/openshift-enterprise/latest/authentication/configuring-internal-oauth.html#customizing-the-oauth-server-url_configuring-internal-oauth

Reply:
@bergerhoffer For [Bug 1992431](https://bugzilla.redhat.com/show_bug.cgi?id=1992431) , Please check [comment 3]( https://bugzilla.redhat.com/show_bug.cgi?id=1992431#c3). For [Bug 1991448](https://bugzilla.redhat.com/show_bug.cgi?id=1991448), notice related lines are added to the WARNING. But I think it's not enough for this bug fix and there should be some fix from Jenkins side based on that. @slaskawi  Is it true?

Reply:
LGTM

Reply:
@yaoli-redhat Since we have a separate doc BZ open for that issue, can we move forward with this initial PR as is, and then use that doc BZ to adjust the wording for this as necessary?

Reply:
LGTM